lib/hl7/message.ex

```diff
@@ -508,18 +508,15 @@ defmodule HL7.Message do
     end
   end
 
-  defp validate_text(raw_text, options) do
-    encoded_text =
-      if options[:accept_latin1] == true do
-        :unicode.characters_to_binary(raw_text, :latin1)
-      else
-        raw_text
-      end
+  defp validate_text(raw_text, %{accept_latin1: true}) do
```

Comment:
I moved this into its own clause since there should be no need to validate after a transform (if one happens).